sdk/python/feast/type_map.py

```diff
@@ -326,13 +326,17 @@ def _python_value_to_proto_value(
                     ProtoValue(unix_timestamp_list_val=Int64List(val=ts))  # type: ignore
                     for ts in int_timestamps_lists
                 ]
-
-            return [
-                ProtoValue(**{field_name: proto_type(val=value)})  # type: ignore
-                if value is not None
-                else ProtoValue()
-                for value in values
-            ]
+            # TODO: Make this better.
+            val_list = []
+            for value in values:
+                if value.dtype == "bool":
+                    value = [bool(e) for e in value]
+                    val_list.append(ProtoValue(**{field_name: proto_type(val=value)}))  # type: ignore
+                elif value is not None:
+                    val_list.append(ProtoValue(**{field_name: proto_type(val=value)}))  # type: ignore
+                else:
+                    val_list.append(ProtoValue())  # type: ignore
+            return val_list
```

Comment:
FYI, I added this in to get some of my tests to pass but I'm not 100% confident that it is actually correct for the larger codebase. Running against the integration tests on master using Github CI to see if it breaks anything. JIC this looks suspicious. same below

Reply:
Yeah def looks suspicious.  Seems like it should maybe be fixed in PYTHON_LIST_VALUE_TYPE_TO_PROTO_VALUE

Reply:
would rather keep it in its original state and leaving maybe a TODO in that PYTHON_LIST_VALUE_TYPE_TO_PROTO_VALUE block to fix boolean lists.